### DISCUSSION TITLE
Is this possible in React Navigation 6?

### DISCUSSION THREAD

**Discussion by damian-balas** [react-navigation/react-navigation#10162] [Q&A]
Hi! :) 

I need to make a drawer menu that is multiple levels deep. To be precise here is an example:

1. We open the drawer menu
2. We click on "settings"
3. We are still in the MENU, but we are redirected to a second "screen". Let's call it "Notification settings"
4. Then we are able to go one more level deeper - to the "User notification settings"
5. We press the back button (can be custom made) and go back to the "Notification settings screen"
6. We are in "settings"
7. We press the back button again and we are in the 1st level Menu again.
8. If we press the back button one more time, then the drawer closes.

Drawer > settings > notification settings > user notification settings
user notification settings > (back) > notification settings > (back) > settings > (back) > drawer > (back) > drawer closes

I don't know how to make this work. Any ideas?

**Comment by pke** [comment]
Can you please change the title of this Q&A to a summary what you want to achieve "How to create a nested drawer navigator"?

I think you would need to handle all this inside the drawer itself, using a custom drawer content.
